### PR TITLE
Expose Tap ring size in env

### PR DESCRIPTION
**Pull request by sknat**
both calico-agent and vpp-manager now take `CALICOVPP_TAP_RING_SIZE` that can be either 
* an int specifying the tap ring buffer size (both RX&TX)
* two ints (RxSize,TxSize) separated by a comma
Default size is 1024

### CODE REVIEW

File: calico-vpp-agent/cni/network_vpp.go [hunk 1]
```diff
@@ -414,6 +414,8 @@ func (s *Server) AddVppInterface(args *pb.AddRequest, doHostSideConf bool) (ifNa
 		MacAddress:     vppSideMacAddress,
 		HostMacAddress: containerSideMacAddress,
 		RxQueues:       config.TapRXQueues,
+		RxRingSize:     config.TaxRxRingSize,
+		TxRingSize:     config.TaxTxRingSize,
```
Comment (AloysAugustin) [inline calico-vpp-agent/cni/network_vpp.go:418]: Typo?

Comment (sknat) [reply calico-vpp-agent/cni/network_vpp.go:418]: Indeed, didn't see it at first glance, thanks 😄

Comment (sknat) [reply calico-vpp-agent/cni/network_vpp.go:418]: I'll do the taxreform then 😛

Comment (AloysAugustin) [reply calico-vpp-agent/cni/network_vpp.go]: 😃

**Pull request merged**